deployment/common/changeset/set_config_mcms.go

```diff
@@ -0,0 +1,207 @@
+package changeset
+
+import (
+	"errors"
+	"fmt"
+	"math/big"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/config"
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/gethwrappers"
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/proposal/mcms"
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/proposal/timelock"
+	chain_selectors "github.com/smartcontractkit/chain-selectors"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/common/proposalutils"
+	commontypes "github.com/smartcontractkit/chainlink/deployment/common/types"
+)
+
+type ConfigPerRole struct {
+	Proposer  config.Config
+	Canceller config.Config
+	Bypasser  config.Config
+}
+type ProposalConfig struct {
```

Comment:
does this struct not already exist?

Reply:
We had it on the example changeset, but I'll have it again here since this one is not an example, and will rename it `TimelockConfig` since this only relates to timelock stuff.